src/main/java/io/vertx/core/http/impl/Http2ServerRequestImpl.java

```diff
@@ -414,9 +409,7 @@ public String getHeader(CharSequence headerName) {
   public MultiMap params() {
```

Comment:
synchronized (conn) should be wrap the whole block we don't really need double checked locking here